lib/fog/azurerm/docs/storage.md

```diff
@@ -176,42 +166,119 @@ result = container.destroy
 puts "#{result}"
 ```
 
-### Metadata
+## Upload a local file as a blob
+```ruby
+new_blob = azure_storage_service.upload_block_blob_from_file('<container name>', '<blob name>', '<file path>')
+puts "#{new_blob.inspect}"
+```
+
+## Download a blob to a local file
+```ruby
+blob = azure_storage_service.download_blob_to_file('<container name>', '<blob name>', '<file path>')
+puts "#{blob.inspect}"
+puts "File Size: #{File.size <file_path>}"
+```
+
+## Delete the storage blob
+
+Mark the specified blob for deletion. The blob is later deleted during garbage collection.
+
+```ruby
+blob = azure_storage_service.blobs(container_name: '<container name>').get('<blob name>')
+result = blob.destroy
+puts "#{result}"
+```
+
+Note that in order to delete a blob, you must delete all of its snapshots.
+
+```ruby
+blob = azure_storage_service.blobs(container_name: '<container name>').get('<blob name>')
+result = blob.destroy(delete_snapshots: 'only')
```

Comment:
options for delete_snapshots are little confusing. e.g. the 'only' options gives impression that we should ONLY delete snapshots and not the blob.

I suggest that we use boolean values for 'delete_snapshots'. When its 'true' snapshots are deleted otherwise not.

What you think?